website/pages/how-to-guides/simple-analytics-to-bigquery.mdx

```diff
@@ -0,0 +1,288 @@
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Authenticate to Google Cloud
+        uses: 'google-github-actions/auth@v0'
+        with:
+          workload_identity_provider: 'projects/${{ secrets.SA_BIGQUERY_PROJECT_NUMBER }}/locations/global/workloadIdentityPools/cloudquery-pool/providers/cloudquery-provider'
+          service_account: 'cloudquery-service-account@${{ secrets.SA_BIGQUERY_PROJECT_ID }}.iam.gserviceaccount.com'
+      - uses: cloudquery/setup-cloudquery@v3
+        name: Setup CloudQuery
+        with:
+          version: "v2.3.1"
+      - name: CloudQuery Sync
+        run: cloudquery sync --log-console simple-analytics.yml bigquery.yml
+        env:
+          SA_USER_ID: ${{ secrets.SA_USER_ID }}
+          SA_API_KEY: ${{ secrets.SA_API_KEY }}
+          SA_BIGQUERY_PROJECT_ID: ${{ secrets.SA_BIGQUERY_PROJECT_ID }}
+          SA_BIGQUERY_DATASET_ID: ${{ secrets.SA_BIGQUERY_DATASET_ID }}
+```
+
+This configuration allows us to run the workflow manually or on a schedule. The example above is configured to run daily at 2am UTC. The workflow will run on a self-hosted runner, which is a virtual machine hosted by GitHub. The workflow will run the following steps:
+ 1. Check out the repository
+ 2. Authenticate to Google Cloud using the [google-github-actions/auth](https://github.com/google-github-actions/auth) Action
+ 3. Setup CloudQuery using the [cloudquery/setup-cloudquery](https://github.com/cloudquery/setup-cloudquery) Action
+ 4. Run the `cloudquery sync` command to load data from Simple Analytics into BigQuery
+
+We will now configure the secrets and the configuration files.
+
+### Step 1.2: Add the secrets
+
+In the GitHub repository, go to `Settings > Secrets` and add the following repository secrets:
+
+- `SA_USER_ID`: the user ID of your Simple Analytics account. You can find this in the Simple Analytics [Account Settings](https://simpleanalytics.com/account).
+- `SA_API_KEY`: the API key of your Simple Analytics account. You can find this in the Simple Analytics [Account Settings](https://simpleanalytics.com/account).
+- `SA_BIGQUERY_PROJECT_ID`: the ID of the Google Cloud project where your BigQuery dataset is located. You can find this in the Google Cloud Console home page when your project is selected.
+- `SA_BIGQUERY_DATASET_ID`: the ID of the BigQuery dataset where you want to load the data. You can find this in the BigQuery UI.
+- `SA_BIGQUERY_PROJECT_NUMBER`: the number of the Google Cloud project where your BigQuery dataset is located.  You can find this in the Google Cloud Console home page when your project is selected.
+
+### Step 1.3: Add the Simple Analytics source plugin configuration file
+
+In the git repository, create a new file called `simple-analytics.yml` and add the following content:
+
+```yaml copy
+kind: source
+spec:
+  name: "simple-analytics"
+  path: "simpleanalytics/simple-analytics"
```

Comment:
This repo doesn't actually exist yet, right now it's hosted under `cloudquery/simple-analytics`